### PR TITLE
Update headers and remove non-v4 references

### PR DESCRIPTION
**Pull request by rhc54**
Signed-off-by: Ralph Castain <rhc@pmix.org>

### CODE REVIEW

**Comment by rhc54** [comment]
bot:ibm:prrte:retest

**Comment by rhc54** [comment]
@jjhursey I'm not sure what is going on here with the IBM/PRRTE CI. The pmix-tests repo has been correctly updated, but the CI keeps running the now-out-of-date version of the test.

Any suggestions?

**Comment by jjhursey** [comment]
looking 👀

**Comment by jjhursey** [comment]
It looks like a stale file on the CI server prevented the update. I cleared it out, and it should work now.
bot:ibm:prrte:retest

File: test/simple/simptool.c [hunk 1]
```diff
@@ -63,6 +63,7 @@ int main(int argc, char **argv)
         pmix_output(0, "Client ns %s rank %d: PMIx_Query_info failed: %d", myproc.nspace, myproc.rank, rc);
         goto done;
     }
+    pmix_output(0, "QUERY COMPLETE");
```
Comment (jjhursey) [inline test/simple/simptool.c:66]: Debug output that should be removed.

**Review by jjhursey — CHANGES_REQUESTED** [review]
Looks good other than that one line.

File: test/simple/simptool.c (hunk 1 above)
Comment (rhc54) [reply test/simple/simptool.c]: removed

**Comment by jjhursey** [comment]
The cross-version make check looks like a timing issue
 * https://gist.github.com/ibm-ompi/152e95c6d2b41f99ad9fd33f7fb070be

```
-----> : Run simptest Make Check
Make Check: /home/pmixer/scratch/pmix-v3.2/test/pmix_client
Server : /home/pmixer/scratch/pmix-hdrs/test
Command: cd /home/pmixer/scratch/pmix-hdrs/test ; timeout --preserve-status -k 35 30 ./pmix_test -n 4 --job-fence -e /home/pmixer/scratch/pmix-v3.2/test/pmix_client
Status : 143 ***FAILED***
-----> : Test output

-----> : Run simptest Make Check
```

bot:ibm:xvercheck:retest

**Review by jjhursey — APPROVED** [review]

**Pull request merged**